mwdb/web/src/components/RecentView/RecentView.js

```diff
@@ -31,6 +31,7 @@ export default function RecentView(props) {
     // Query error shown under the query bar
     const [queryError, setQueryError] = useState(null);
     const [objectCount, setObjectCount] = useState(null);
+    const [getObjectCount, setGetObjectCount] = useState(true);
```

Comment:
I would name it a bit better
```suggestion
    const [countingEnabled, setCountingEnabled] = useState(true);
```